### DISCUSSION TITLE
wifi stuck at ''starting wifi association''

### DISCUSSION THREAD

**Discussion by dannler** [Guilouz/Klipper-Flsun-Speeder-Pad#58] [General]
i was initially connected to a wifi network and opted to change to another, since then it doesnt connect and stalls at "starting wifi association"  top of network screen it says interface:eth0

how do i reconnect to my network again, countless of system restarts and connecting to network doesnt help at all

**Comment by SckyzO** [comment]
Delete old network. Reboot, check if it has been deleted. And reconnect to new Wifi

**Comment by OrByR** [comment]
Connect to pad via UART (115200)
sudo nano /etc/wpa_supplicant/wpa_supplicant.conf
Clean out all the listed neworks but yours and remove the disabled=1 line on yours if it`s there.
Save and reboot you Speeder Pad.
